.github/workflows/create-release.yml

```diff
@@ -32,8 +32,8 @@ jobs:
       - name: Check image Tag
         run: ./scripts/check_tag_info.sh ${{ github.event.inputs.name }}
 
-      - name: Verify prow post jobs
-        run: ./scripts/verify-status.sh ${{ github.ref_name }}
+      - name: Verify job status
+        run: ./scripts/verify-status.sh ${{ github.ref_name }} 600 10 45
```

Comment:
I think we do not need to wait, here.

I think we need this check here at all. There will not be any post jobs on the release branch and we only really need to check the release build job, further down.

Reply:
We don’t know the behaviour of the check. In this case we have chance the check it manually